packages/cf.js/API_REFERENCE.md

```diff
@@ -0,0 +1,54 @@
+# API V0.0.2
+## `cf.js`
+
+- `Client`
+    - Properties
+        - `nodeProvider`
+    - Instance methods
+        - `getAppInfos(): AppInfo[]`
+        - `createAppFactory(appDefinition: AppDefinition): AppFactory`
```

Comment:
Will the `AppFactory` that creates an instance of its app implicitly give the app a handle on the `nodeProvider` from the client so that the app can talk to the node?

Reply:
Seems odd to have this method versus using the 'new' operstor. @ebryn

Reply:
Yes to what @cf19drofxots said. It's somewhat odd but less odd than passing the injected `nodeProvider` to each instance of AppFactory and AppInstance to communicate with the node.

Reply:
@ebryn Do you have any input here?